Bluepill-cli/BPInstanceTests/BPTestHelper.m

```diff
@@ -41,6 +41,11 @@ + (NSString *)sampleAppFatalTestsBundlePath {
 + (NSString *)resourceFolderPath {
     return [[NSBundle bundleForClass:[self class]] resourcePath];
 }
+
++ (NSString *)bpExecutablePath {
```

Comment:
what @ob means is we can use this logic in `BPRunner.m:BPRunnerForApp` and then we don't need to set up some environment variable